lib/TextOperations.cpp

```diff
@@ -210,4 +248,28 @@ std::string TextOperations::escapeString(std::string input)
 	return input;
 }
 
+int TextOperations::extractNumberAfterHash(const std::string & input)
+{
+    size_t hashPos = input.find('#');
+	if(hashPos != std::string::npos)
+		return -1;
+
+	std::string numberAsString = input.substr(hashPos + 1); // Get the substring after '#'

```

Comment:
Only one digit??

Reply:
Bug!
`std::string numberAsString = input.substr(hashPos + 1, input.length() - (hashPos + 1)); // Get the substring after '#' until the end of the string`

Reply:
Bug!
`std::string numberAsString = input.substr(hashPos + 1, input.length() - (hashPos + 1)); // Get the substring after '#' until the end of the string`